wbstack/src/Settings/LocalSettings.php

```diff
@@ -235,11 +235,6 @@
     ];
 }
 
-// Output compression needs to be disabled in 1.35 until the below phab task is fixed...
-// TODO dig more to see if there is something else to do here...
-// https://phabricator.wikimedia.org/T235554
-$wgDisableOutputCompression  = true;
```

Comment:
This issue seems to be fixed, I tested it locally with something like http://potatopotato.wbaas.localhost/wiki/Special:EntityData/Q1.json and it returned raw JSON